server/publications.coffee

```diff
@@ -42,3 +42,7 @@ Meteor.publish 'articles', (query={}) ->
 
 Meteor.publish 'feeds', ->
   Feeds.find()
+
+# User status
+Meteor.publish 'userStatus', () ->
+  Meteor.users.find({'status.online': true }, {fields: {'profile': 1, 'status': 1 }, transform: null})
```

Comment:
I was thinking about doing the following with profile.name but did not.

![image](https://cloud.githubusercontent.com/assets/12954045/21022472/bf48ad6e-bd4a-11e6-970a-fb943c6befec.png)

The `transform: null` can increase the speed of the find operation by avoiding transformation layers by 3rd party plugins. In my experience, this is extremely noticeable when a ORM is used, which I'm not sure if we're using one or not in the project, so I say this is defensive programming on my part.